src/lib/structures/ExtendedArgument.ts

```diff
@@ -44,7 +44,7 @@ import { Argument, ArgumentContext, ArgumentOptions, ArgumentResult, AsyncArgume
  *   handle(parsed, { argument }) {
  *     return parsed.type === 'text'
  *       ? this.ok(parsed)
- *       : this.error(argument, 'ArgumentTextChannelInvalidTextChannel', 'The argument did not resolve to a text channel/');
+ *       : this.error({ identifier: 'ArgumentTextChannelInvalidTextChannel', message: 'The argument did not resolve to a text channel' });
```

Comment:
Too long, please format this line with Prettier so it fits within the character count.